### PR TITLE
Add new reserved keyword

### PR DESCRIPTION
**Pull request by olivierdet**
Using `pull_reserved_words.sh`, this add new reserved keywords. 
In my project, table creation with column named `pivot` was failing.

## Todos
- [ ] MIT compatible
- [ ] Tests
- [ ] Documentation
- [ ] Updated CHANGES.rst

### CODE REVIEW

File: sqlalchemy_redshift/dialect.py [hunk 1]
```diff
@@ -154,30 +154,31 @@ class RedshiftImpl(postgresql.PostgresqlImpl):
...
+    "encode", "encrypt", "encryption", "end", "except", "explicit", 
+    "false", "for", "foreign", "freeze", "from", "full", "globaldict256", 
+    "globaldict64k", "grant", "group", "gzip", "having", "identity", 
+    "ignore", "ilike", "in", "initially", "inner", "intersect", "into", 
+    "is", "isnull", "join", "language", "leading", "left", "like", 
+    "limit", "localtime", "localtimestamp", "lun", "luns", "lzo", "lzop", 
+    "minus", "mostly16", "mostly32", "mostly8", "natural", "new", "not", 
+    "notnull", "null", "nulls", "off", "offline", "offset", "oid", "old", 
+    "on", "only", "open", "or", "order", "outer", "overlaps", "parallel", 
+    "partition", "percent", "permissions", "pivot", "placing", "primary",
```
Comment (jklukas) [inline sqlalchemy_redshift/dialect.py:174]: There aren't any differences until this line. Do you know why lines 157 through 173 show up in the diff? Is there some change in whitespace here?

**Review by jklukas — CHANGES_REQUESTED** [review]
Thanks for making the PR! This looks good, expect that I'd like to keep the diff clean, so we need to understand why all these lines look like they're changing. Perhaps there's trailing spaces introduced?

File: sqlalchemy_redshift/dialect.py (hunk 1 above)
Comment (olivierdet) [reply sqlalchemy_redshift/dialect.py]: I've seen that there were trailing space (due to copy pasting I suppose). I've removed them. 
The diff is cleaner now. :-)

**Review by jklukas — APPROVED** [review]
Looks great! Sorry it took so long to get to reviewing.

**Pull request merged**